### DISCUSSION TITLE
The offset feature in the filter... -300 to +300 WHAT?

### DISCUSSION THREAD

**Discussion by UnbridledGames** [jensrossbach/node-red-contrib-chronos#55] [Q&A] [answered]
I've looked everywhere. All I can see is a range of - to + 300 on the offset... but +/- 300 WHAT? Seconds? Minutes? Hours? Days? Millenia?

I honestly don't have the time (ha!) to test this and try and figure it out, But the units used is sort of important.

**Comment by jensrossbach** [comment, marked as answer]
Hi, the unit of the offset is minutes as documented on the help pages as well as in the wiki.

[Time Filter Node - Operands](https://github.com/jensrossbach/node-red-contrib-chronos/wiki/Time-Filter-Node#operands)
